client/src/BookingTimeRangeSelector.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, FC } from "react";
+import { Typography, Card, CardContent, TextField } from "@mui/material";
+import AdapterDateFns from "@mui/lab/AdapterDateFns";
+import { LocalizationProvider, StaticDatePicker, TimePicker } from "@mui/lab";
+
+type Props = {
+  title: string;
+};
+export const TimeSetter: FC<Props> = ({ title }) => {
```

Comment:
Well done on deciding to split this into two components! It is however recommended to have one FC per file, so you can split this into two smaller components: TimeSetter.tsx and DateSetter.tsx.